content/en/blog/2019/app-identity-and-access-adapter/index.md

```diff
@@ -0,0 +1,167 @@
+		paths:
+		- prefix: /webapp
+			method: ALL
+			policies:
+			- policyType: oidc
+				config: my-oidc-provider-config
+				rules: // optional
+				- claim: iss
+					match: ALL
+					source: access_token
+					values:
+					- <expected-issuer-id>
+				- claim: scope
+					match: ALL
+					source: access_token
+					values:
+					- openid
+```
+
+[Read more about protecting web applications](https://github.com/ibm-cloud-security/app-identity-and-access-adapter)
+
+## Protecting backend application and APIs
+
+Backend applications and APIs are protected using the Bearer Token flow, where an incoming token is validated against a particular policy. The Bearer Token authorization flow expects a request to contain the `Authorization` header with a valid access token in JWT format. The expected header structure is `Authorization: Bearer {access_token}`. In case token is successfully validated request will be forwarded to the requested service. In case token validation fails the HTTP 401 will be returned back to the client with a list of scopes that are required to access the API.
+
+### Applying backend application and APIs protection
+
+Protecting backend applications and APIs requires creating two types of resources - use `JwtConfig` resources to define various JWT providers, and `Policy` resources to define the backend app protection policies. 
+
+```yaml
+apiVersion: "security.cloud.ibm.com/v1"
+kind: JwtConfig
+metadata:
+	name: my-jwt-config
+	namespace: sample-namespace
+spec:
+	jwksUrl: <the-jwks-url>
+```
+
+```yaml
+apiVersion: "security.cloud.ibm.com/v1"
+kind: Policy
+metadata:
+	name: my-sample-backend-policy
+	namespace: sample-namespace
+spec:
+	targets:
+	- serviceName: <kubernetes-service-name-to-protect>
+		paths:
+		- prefix: /api/files
+			method: ALL
+			policies:
+			- policyType: jwt
+				config: my-oidc-provider-config
+				rules: // optional
+				- claim: iss
+					match: ALL
+					source: access_token
+					values:
+					- <expected-issuer-id>
+				- claim: scope
+					match: ALL
+					source: access_token
+					values:
+					- files.read
+					- files.write
+```
+
+[Read more about protecting backend applications](https://github.com/ibm-cloud-security/app-identity-and-access-adapter)
+
+## Known limitations
+
+At the time of writing this blog there are two known limitations of the App Identity and Access adapter:
+
+- Due to the way Envoy Proxy was handling HTTP headers it was impossible to return multiple `Set-Cookie` headers from Mixer back to Envoy. Therefore we couldn't set all the cookies required for handling Web Application scenarios. As a result, if you use the App Identity and Access adapter for Web Applications you should not use more than a single replica of the adapter. The issue was recently addressed in Envoy and Mixer and we're planning to address this in future versions of our adapter. **Note that this only affects Web Applications, and doesn't affect Backend Apps and APIs in any way**. 
```

Comment:
I would start the bullet with this sentence instead, using the pattern of stating the action to take and providing the reason after it.